docs/commands/otp.md

```diff
@@ -17,6 +17,7 @@ Note: If `show.safecontent` is enabled, OTP URLs are hidden from the `show` comm
 | Flag         | Aliases | Description                                                              |
 |--------------|---------|--------------------------------------------------------------------------|
 | `--clip`     | `-c`    | Copy the time-based token into the clipboard.                            |
+| `--autoclip` | `-C`    | Copy the time-based token into the clipboard and show it.                |
```

Comment:
```suggestion
| `--alsoclip` | `-C`    | Copy the time-based token into the clipboard and show it.                |
```

As mentioned by @dominikschulz  we shouldn't have two different names in `show` vs `otp` for the same functionality and it seems you've changed it below already.

Reply:
Yes, I would stick with `alsoclip` for the sake of this PR.
If we want to rename it we should be doing that in isolation, i.e. in a PR that only does the rename so we can revert it easily, if needed.